nextflow.config

```diff
@@ -30,10 +30,27 @@ profiles {
     params.validation_script = "dynamic_challenge_validate.py"
     params.email_script = "dynamic_challenge_send_email.py"
   }
-  pegs_challenge {
-    params.view_id = "syn53770151"
-    params.input_id = "syn51390589"
-    params.scoring_script = "model_to_data_score.py"
+  pegs_challenge_validate {
+    params.project_name = "PEGS DREAM Challenge - staging"
+    params.view_id = "???"
```

Comment:
I don't have access to the submission views

Reply:
Can you use this one? syn57373526

Reply:
Sure thing! Do you also have one for the test round?

Reply:
Just created one here: syn58942525

(For now, I'm keeping this submission view hidden from the public, since we don't want to share results with anyone (including the submitters) until after the challenge has concluded and top performers are determined).

Reply:
Updated!